core/src/test/java/bio/terra/pearl/core/service/survey/SurveyResponseServiceTests.java

```diff
@@ -138,20 +143,32 @@ public void testSurveyResponseUpdateAnswers(TestInfo testInfo) {
 
         List<DataChangeRecord> changeRecords = dataChangeRecordService.findByEnrollee(savedResponse.getEnrolleeId());
         assertThat(changeRecords.size(), equalTo(1));
-        assertThat(changeRecords.get(0), samePropertyValuesAs(DataChangeRecord.builder()
```

Comment:
`samePropertyValuesAs` was erroring on nulls even if I excluded the null fields from being checked. What's odd is that there were already null properties being checked in the existing test, so I'm not sure what's going on here. I spent some time in the debugger trying to figure out what was going on and was completely stumped- everything was identical